app/src/main/java/ch/hikemate/app/ui/auth/CreateAccountScreen.kt

```diff
@@ -114,11 +120,12 @@ fun CreateAccountScreen(
       modifier =
           Modifier.testTag(Screen.CREATE_ACCOUNT)
               .padding(
-                  // Add for the status bar
                   start = 16.dp,
                   end = 16.dp,
-                  top = 40.dp,
-              ),
+              )
+              .safeDrawingPadding()
```

Comment:
Praise: this is a simple addition, but the kind of simple that takes hours to find. Very nice!